samples/app-identity-linking-with-sso/nodejs/views/tab.ejs

```diff
@@ -198,34 +198,35 @@
                         mode: 'cors',
                         cache: 'default'
                     })
-                        .then((response) => {
-                            if (response.ok) {
-                                return response.json();
-                            } else {
-                                reject(response.error);
+                    .then((response) => {
+                        if (response.ok) {
+                            return response.json();
+                        } else {
+                            reject(response.error);
+                        }
+                    })
+                    .then((responseJson) => {
+                        if (responseJson.error) {
+                            reject(responseJson.error);
+                        } 
+                        else {
+                            const userDetails = responseJson;
+                            $("#sso").hide();
+                            $("#userImgSso").attr("src", userDetails.image);
```

Comment:
Naming conversions are not correct for element id's like 
$("**#sso**").hide();
$("**#work**").append(userDetails.details.jobTitle);